README.md

```diff
@@ -1 +1,9 @@
 # cs-5666-scatt-gp
+
+## Demo Instructions
+* Login to the Appalachian student server.
```

Comment:
@haietza Ah, I didn't think about that!  You are right.  I also tried running make scatt on student, then pulling the Scatt.jar down to run it from my desktop.  It runs, but if after entering a folder name it throws the "Class definition not found" for the JSON parser, meaning it's missing that dependency somehow.  I don't know much at all about .jar files, I think the issue you have already created can be used for figuring this out!